script/update-locales.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Format and update translations from a source file (en_US.json by default)
+#
+# Requires jq (https://stedolan.github.io/jq/) and sponge
+# (https://www.putorius.net/linux-sponge-soak-up-standard-input-and-write-to-a-file.html)
```

Comment:
@clangen Do you find introducing these utilities as a developer dependency acceptable?